### PR TITLE
TASKLESS Add rhoc shortcuts for stopping and deleting connectors

### PR DESCRIPTION
**Pull request by rk3rn3r**

### CODE REVIEW

**Comment by rk3rn3r** [comment]
wdyt @lburgazzoli @rinaldodev ?

File: bin/get-connectors-with-clusters [hunk 1]
```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
-CLUSTERS=$(get-api kafka_connector_clusters | jq '.items')
-NAMESPACES=$(get-api kafka_connector_namespaces | jq '.items')
-CONNECTORS=$(get-api kafka_connectors | jq '.items')
+BASE_PATH=$(dirname -- "$( readlink -f -- "$0"; )";)
```
Comment (rk3rn3r) [inline bin/get-connectors-with-clusters:3]: Using a relative base path makes the command executable out of this direnv dirs that are outside of the rhoc bin dir.

**Comment by lburgazzoli** [comment]
@rinaldodev @valdar can you take a look ?

**Review by rinaldodev — APPROVED** [review]

**Comment by lburgazzoli** [comment]
@rk3rn3r I forgot to merge this PR, is there anything more you want to add or ready to merge ?

**Comment by rk3rn3r** [comment]
@lburgazzoli not problem. It's just some enhancement. gtm

**Pull request merged**